rollup.config.js

```diff
@@ -14,9 +14,10 @@ const globals = {
   'prop-types': 'PropTypes',
 };
 
-const getBabelOptions = () => ({
+const getBabelOptions = ({ useESModules }) => ({
   exclude: '**/node_modules/**',
   runtimeHelpers: true,
+  plugins: [['@babel/transform-runtime', { polyfill: false, useBuiltIns: true, useESModules }]],
 });
```

Comment:
Should we test that it's still working with all the browsers we support. Like IE 11, does it handle `Object.assign`?

Reply:
Test your latest release. It's the same.

Reply:
How is that the same?
```js
var _Object$assign = _interopDefault(require('babel-runtime/core-js/object/assign'));
``` 
vs
```js
Object.assign
```

Reply:
Merge [this](https://github.com/oliviertassinari/react-event-listener/pull/74) nice PR to solve the problem.